features/buyer/buyer_creation.feature

```diff
@@ -0,0 +1,27 @@
+@newbuyer @skip-staging @notify
+Feature: Create buyer account. We're assuming a user journey where someone tries to find a specialist, eventually hits the login page, and then clicks 'Create a buyer account' from the login page
```

Comment:
I don't think we need the second sentence here - the flow is clear from the steps taken. It feels like the "Feature" and "Scenario" are the same thing here, but if we want to be specific then could go with something like:
```
Feature: Create a buyer account

Scenario: Create a new buyer account when trying to create a requirement from the home page
```

Reply:
That's a very good point to put additional details in scenario rather than feature, then we could potentially have different journey for buyer to create account in the same feature in the future :).